modules/rtdModule/index.js

```diff
@@ -128,6 +128,31 @@ function getProviderData(adUnits, callback) {
   }
 }
 
+/**
+ * delete invalid data received from provider
+ * this is to ensure working flow for GPT
+ * @param {Object} data received from provider
+ * @return {Object} valid data for GPT targeting
+ */
+function validateProviderDataForGPT(data) {
+  // data must be an object, contains object with string as value
+  if (typeof data !== 'object') {
+    return {};
+  }
+  for (let key in data) {
+    if (data.hasOwnProperty(key)) {
+      for (let innerKey in data[key]) {
+        if (data[key].hasOwnProperty(innerKey)) {
+          if (typeof data[key][innerKey] !== 'string') {
+            delete data[key][innerKey];
```

Comment:
Dropping data without a warning that data is being dropped seems problematic to me

Reply:
added warning